modules/500-upmeter/images/upmeter/src/pkg/probe/checker/converge.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package checker
+
+import (
+	"time"
+
+	"d8.io/upmeter/pkg/check"
+	"github.com/sirupsen/logrus"
+)
+
+type convergeStatusChecker struct {
+	history             []status
+	startupConvergeDone bool
+	historyCleaned      bool
+	poller              poller
+	logger              *logrus.Entry
+
+	windowSize                  int
+	period                      time.Duration
+	freezeThreshold             time.Duration
+	allowedTasksPerTimeInterval float64
```

Comment:
Seems `allowedTasksPerTimeInterval` is not used

Reply:
it is used (here)[https://github.com/deckhouse/deckhouse/blob/fix/upmeter-lifecycle-deckhouse/modules/500-upmeter/images/upmeter/src/pkg/probe/group_deckhouse.go#L87]

Reply:
oh, wait, you're right, it's not used in this file

Reply:
fixed